### PR TITLE
Added argwhere function to jax.numpy namespace functions and passed tests. #6049

### PR DESCRIPTION
**Pull request by waseef3277**
argwhere #6049

### CODE REVIEW

**Comment by waseef3277** [comment]
Please can someone review my PR?

**Comment by waseef3277** [comment]
> Hello @waseef3277, It seems like you're trying to modify an existing frontend function already, could you revert that and instead add a new function? also could you resolve the conflict? 

It was not there when I created an issue #6049, you can have a look.

**Comment by waseef3277** [comment]
> > It was not there when I created an issue #6049, you can have a look.
> 
> Maybe it was added while you were working on this function, your changes override the function, I'd resetting changes and readding your function to the recent commit from master, this will also resolve the branch conflict, thanks!
> 

Okay please do that, I will have to add another function after that?

**Comment by CatB1t** [comment]
> > > It was not there when I created an issue #6049, you can have a look.
> > 
> > 
> > Maybe it was added while you were working on this function, your changes override the function, I'd resetting changes and readding your function to the recent commit from master, this will also resolve the branch conflict, thanks!
> 
> Okay please do that, I will have to add another function after that?

Sorry for the confusion, I meant to say *_I'd suggest to reset the changes_, you have to do to it from your side.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
